hack/RELEASE.md

```diff
@@ -0,0 +1,77 @@
+     or file content changes required by automation.  However, it *may* contain
+     any necessary code changes not otherwise present in the PR.
+
+   * The commit *must* be GPG signed with a white-listed public key contained
+     in the file at `https://FIXME.com/FIXME/FIXME_WHITELIST.txt`
+
+2. Automation *must not* be sensitive to the presence/absence of any other commits
+   contained in the release PR, beyond HEAD.  However, all commits *must* pass
+   automated testing *and* be fast-forward merge-able to the relevant branch.
+
+   * Updates to the following files (and/or their future equivalents),  *shall* be
+     present in a commit in the PR:
+
+     1. `version/version.go`
+     2. `contrib/spec/podman.spec.in`
+     3. The result of executing `make changelog`
+
+   * Under normal conditions, automation *will* provide feedback to the PR in terms of
+     any failures and/or detectable problems. An unlimited number of opportunities will
+     be provided to re-test the PR with or without corresponding commit changes. Feedback
+     *may* be provided as to the suitability of the PR as a new project release.
+
+4. When deemed ready, the release PR is merged by humans or automation, and *must* (again)
+   successfully pass all automated testing on the branch.
+
+   * Upon failure and/or detectable abnormal conditions, notifications will be made to the
+     #podman channel on Freenode IRC *and* by e-mail to the recipients specified in the file
+     at `https://FIXME.com/FIXME/FIXME_EMAIL.txt`
+
+   * An unlimited number of opportunities *will* be provided by automation, to re-test the
+     branch.  In all cases, feedback *will* be provided as to the fitness of the automated
+     release workflow for the existing content.
+
+   * No further branch commits will be considered for the new release.  If changes are needed
+     (for example to correct an undetected bug), the process must begin again with the next
+     symentic version.  ***Note:*** In this special case, the `Prior Release` reference text
+     of all future commits *must not* reference the errant version.
+
+5. Assuming branch-testing passes automated checks.
+
+   * A new tag and branch *will* be created by automation, referencing the "Release vX.Y.Z"
+     commit.
+
+   * The tag *will* be named like `vX.Y.Z[-extra]' and the branch *will* be named `vX.Y`.
+
+   * The tag *will* be GPG signed by automation using a key accepted by the core maintainers.
+     The tag annotation (message) *shall* be a string of the following format:
+     ``<prior release> -> <tag name> from <originating branch>``.
+
+   * Automation *may* create a Github-Release object, reflecting the name of the tag.  All
+     available build-time artifacts from branch-testing *may* be uploaded to the object.
+
+
+# Post-release Checklist
+
+0. Go to quat/libpod/gate "builds" page.  Verify a 'gate' image build is running or completed
```

Comment:
This is probably a typo for quay. Why make the reader have to work? Why not include the full url? https://quay.io/repository/libpod/gate?tab=builds

Reply:
good idea, thanks!